docs/walk-through.md

```diff
@@ -257,15 +275,39 @@ $ curl \
     --silent        `# Don't write progress bar to stderr` \
     --header "Content-Type: application/json; odata=minimalmetadata" \
     --data "{'applicationId':'$applicationId', 'token':'$AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN'}" \
-    $AZ_BATCH_ACCOUNT_URL/softwareEntitlements?api-version=$apiVersion \
+    $AZ_BATCH_ACCOUNT_URL/softwareEntitlements?api-version=2017-09-01.6.0 \
     | jq
+```
+
+The output should be similar to:
+```json
+{
+  "id": "entitlement-695c822b-470c-4fd1-b34d-9b21435f57b2",
+  "expiry": "2019-01-16T01:06:07+00:00"
+}
+```
+
```

Comment:
Might be worth adding a snippet to quantify the similarity/difference. 

Perhaps

> The actual `id` value and `expiry` timestamp will differ, but the structure will be the same.

??

Reply:
Added descriptive comments to the fields themselves.